keywords/hashtag.md

```diff
@@ -1,12 +1,18 @@
 ## Hashtag
 
-A hashtag, denoted by the symbol “#”, is a metadata tag commonly use in the Internet culture, particularly on social media. [^Merriam-Webester24Hashtag]  It originated as a feature on a post on Twitterin August 2007 from a product designer named Chris Messina. The primary purpose was to provide user with an easy way to categorize, discover and participate in specific topics on social media.[^Hastags.org12Whatis#]
+A hashtag, denoted by the symbol “#”, is a metadata tag commonly use in the Internet culture, particularly on social media. [^Merriam-Webester24Hashtag]  It originated as a feature on a post on Twitterin August 2007 from a product designer named Chris Messina. The primary purpose was to provide user with an easy way to categorize, discover and participate in specific topics on social media.[^Hastags.org12Whatis#] In addition, Tweets containing hashtags can link to a specific timeline of other tweets with the same hashtag. [^WikströmHashtagOnTwitter]
```

Comment:
@cathytham, there shouldn't be any space between a punctuation mark and citations. Please remove.